setup.py

```diff
@@ -87,7 +87,7 @@
         'ipython_genutils',
         'traitlets>=4.2.1',
         'jupyter_core>=4.4.0',
-        'jupyter_client>=6.1.0',
+        'jupyter_client>=5.3.1',
```

Comment:
Sigh.  Can we please move the floor of jupyter_client to at least `5.3.4` and jupyter_core to `4.6.1`.  Otherwise we'll have the same triage nightmare taking place regarding `secure_write()` that we've been battling the last 3 months in Notebook.

Reply:
Sure, I was just trying to see if it worked.